tests/Tests/100__deploy/100__installation/101__installation.robot

```diff
@@ -33,6 +33,7 @@ Verify User Can Access RHODS Documentation From OCM Console
   [Tags]  ODS-1303
   ...     Tier2
   ...     AutomationBug
```

Comment:
Is `AutomationBug` still valid?

Reply:
hm I think it's safe to remove the label, I see it succeeded last run. Previous failures were labeld as "system issue"